R/build-articles.R

```diff
@@ -20,6 +20,52 @@
 #' "Get started". Rmarkdown files in `vignettes/tutorials/` are ignored,
 #' because these are assumed to contain tutorials, see `build_tutorials()`.
 #'
+#' @section Index and navbar:
+#' You can control the index page and navbar with `articles`, which defines a
+#' list of sections. Each section has four fields:
+#'
+#' * `title` (required): title of section, which appears as a heading on the
+#'   articles index.
+#'
+#' * `desc` (optional): An optional markdown description displayed underneath
+#'   the section title.
+#'
+#' * `navbar` (optional): A couple of words used to label this section in
+#'   the navbar. If omitted, this section of vignettes will not appear in the
+#'   navbar. The navbar will automatically include a link to the articles
+#'   index when not all vignettes are listed individually.
+#'
+#' * `contents` (required): a list of article names to include in the
```

Comment:
Move contents after title so both required are together.

Reply:
I actually started with it there, but then it doesn't reflect the order you usually write the fields in 🤷‍♂